### PR TITLE
Add questions when creating a survey

### PR DESCRIPTION
**Pull request by dipshit**
Screen Shot 2020-03-20 at 1 26 08 AM

Screen Shot 2020-03-20 at 1 26 14 AM

Screen Shot 2020-03-20 at 1 26 22 AM

Screen Shot 2020-03-20 at 1 26 29 AM

L7 FED
Spotlight uuuhhh moonlight uuhhh

### CODE REVIEW

File: client/src/components/QuestionList.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  const handleMinChange = useCallback(newMin => setRangeMin(newMin), []);
+  const handleMaxChange = useCallback(newMax => setRangeMax(newMax), []);
+  const handleTextChange = useCallback(newText => setTextPrompt(newText), []);
+  const handleOptionChange = useCallback(
+    newOptions => setOptions(newOptions),
+    []
+  );
+
+  return questions && questions.length > 0
+    ? questions.map((question, index) => {
```
Comment (vikrambombhi) [inline client/src/components/QuestionList.js]: Turnery operators are head to read we should make this into a if statement latter

Comment (dipshit) [reply client/src/components/QuestionList.js]: just use glasses

Comment (dipshit) [reply client/src/components/QuestionList.js]: "its an elegant way of making a declarative UI" -ismail syed when debating tobi lutke

File: client/src/components/SurveyForm.js [hunk 2]
```diff
@@ -13,15 +13,8 @@ export function SurveyForm() {
...
-        method: "POST",
-        headers: {
-            "Accept": "application/json",
-            "Content-Type": "application/json",
-        },
-        body: JSON.stringify({
-            name: `${name}`,
-        })
+      const res = await fetch(`/api/new?name=${name}`, {
+        method: "POST"
```
Comment (bkzhang) [inline client/src/components/SurveyForm.js]: you uh...reverted this change, I'm guessing the rebase got messed up

**Pull request merged**